src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/generated.py

```diff
@@ -98,8 +98,8 @@
 cli_command('resource policy assignment list', list_policy_assignment)
 cli_command('resource policy assignment show', show_policy_assignment)
 factory = lambda _: _resource_policy_client_factory().policy_definitions
-cli_command('resource policy create', create_policy_definition)
-cli_command('resource policy delete', PolicyDefinitionsOperations.delete, factory)
-cli_command('resource policy list', PolicyDefinitionsOperations.list, factory)
-cli_command('resource policy show', PolicyDefinitionsOperations.get, factory)
-cli_command('resource policy update', update_policy_definition)
+cli_command('resource policy definition create', create_policy_definition)
```

Comment:
The PR just moved all policy definition commands from `az resource policy` to `az resource policy definitions`. So there are no commands under `az resource policy` now, rather just 2 groups, `defintion` and `assignment`. ARM team prefers this layout, and i agree.